pkg/controller/machine/machine_controller.go

```diff
@@ -305,3 +353,26 @@ func (r *ReconcileMachine) deleteNode(ctx context.Context, cluster *clusterv1.Cl
 
 	return corev1Remote.Nodes().Delete(name, &metav1.DeleteOptions{})
 }
+
+// getMachinesInCluster returns all of the Machine objects that belong to the
+// same cluster as the provided Machine
+func (r *ReconcileMachine) getMachinesInCluster(ctx context.Context, namespace, name string) ([]*clusterv1.Machine, error) {
+	if name == "" {
+		return nil, nil
```

Comment:
Return an error if either namespace or name is empty?

Reply:
This is the same behavior as before, so I just kept with that. Is there an existing error we should use?

Reply:
Sorry, had this code mixed up with all the other places in CAPI where we return errors if cluster is missing 😄. I think this is ok. It just means it'll hit the `errNoControlPlaneNodes` branch.